package.json

```diff
@@ -49,10 +47,13 @@
     "node-slackr": "0.1.0",
     "octonode": "^0.7.4",
     "optimist": "0.2.0",
-    "rewire": "^2.3.4",
+    "passport": "^0.3.2",
+    "passport-google-oauth": "^0.2.0",
+    "qs": "3.1.0",
     "request": "2.9.2",
-    "socket.io": "0.8.7",
+    "rewire": "^2.3.4",
```

Comment:
Since we're using this in production perhaps we should pin rewire to 2.3.4 instead of ^2 in case breaking changes get merged in upstream